client/app/(auth)/createProfile.tsx

```diff
@@ -1,19 +1,430 @@
+                                })
+                              : ""
+                          }
+                        />
+                        {errors.dateOfBirth && (
+                          <WarningCircle
+                            color="#dc2626"
+                            size={20}
+                            weight="bold"
+                          />
+                        )}
+                      </View>
+                    </Pressable>
+
+                    <Modal
+                      animationType="slide"
+                      onRequestClose={() => setShowDatePicker(false)}
+                      transparent
+                      visible={showDatePicker}
+                    >
+                      <TouchableWithoutFeedback
+                        onPress={() => setShowDatePicker(false)}
+                      >
+                        <View className="flex-1 items-center justify-end">
+                          <View className="w-full rounded-t-2xl bg-gray-920 p-8">
+                            <DateTimePicker
+                              display="spinner"
+                              maximumDate={new Date()}
+                              mode="date"
+                              onChange={(_, selectedDate) => {
+                                if (selectedDate) {
+                                  setSelectedDate(selectedDate);
+                                  onChange(selectedDate);
+                                }
+                              }}
+                              themeVariant="dark"
+                              value={value ? new Date(value) : new Date()}
+                            />
+                            <Pressable
+                              className="min-h-12 justify-center rounded-md bg-purple-600"
+                              onPress={() => {
+                                if (selectedDate) {
+                                  onChange(selectedDate);
+                                }
+                                setShowDatePicker(false);
+                              }}
+                            >
+                              <Text className="pbk-h6 text-center text-base-white">
+                                DONE
+                              </Text>
+                            </Pressable>
+                          </View>
+                        </View>
+                      </TouchableWithoutFeedback>
+                    </Modal>
+                  </>
+                )}
+              />
+              {errors.dateOfBirth && (
+                <Text className="pbk-b3 mb-4 text-red-600">
+                  {errors.dateOfBirth.message}
+                </Text>
+              )}
+
+              <Text className="pbk-b2 mb-1.5 text-base-white">Avatar</Text>
+              <View className="mb-2 mt-4 flex-row items-center justify-between">
+                <Pressable onPress={() => setShowBottomDrawer(true)}>
```

Comment:
If you have your keyboard open and you open the bottom drawer, the keyboard doesn't seem to dismiss. I think this is because the Pressable captures the touch event and ignores the TouchableWithoutFeedback component. One way to fix this is to explicitly call `Keyboard.dismiss` in the `onPress` of this Pressable. This bug is also there on the "Change avatar" Pressable.

```suggestion
                <Pressable onPress={() => { Keyboard.dismiss(); setShowBottomDrawer(true);}}>
```

Reply:
Nice catch, your solution works great!